src/counter/src/azurefiles/counter/App Resources/network.yaml

```diff
@@ -1,14 +1,14 @@
-## Network definition ##
-network:
-  schemaVersion: 1.0.0-preview1
-  name: counterAppNetwork
-  properties:
-    description: Azure Service Fabric Mesh Counter Application network.
-    addressPrefix: 10.0.0.4/22
-    ingressConfig:
-      layer4:
-        - name: counterServiceIngress
-          publicPort: 8080
-          applicationName: counterApp
-          serviceName: counterService
+## Network definition ##
+network:
+  schemaVersion: 1.0.0-preview2
+  name: counterAppNetwork
+  properties:
+    description: Azure Service Fabric Mesh Counter Application network.
+    addressPrefix: 10.0.0.4/22
+    ingressConfig:
```

Comment:
The network yaml format in preview2 is different. We need to fix that and add Gateway yaml if not added.

Reply:
i have moved back to `preview1` for azurefiles as it requires more changes that @wangcai0124  is working on.
Lets have separate PR for `azurefiles` with all fixes.